integration/kubernetes/runtimeclass_workloads/pod-oom.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       imagePullPolicy: Always
       name: oom-test
       command: ["/bin/sh"]
-      args: ["-c", "sleep 2; stress --vm 2 --vm-bytes 400M --timeout 5s"]
+      args: ["-c", "sleep 2; stress --vm 2 --vm-bytes 400M --timeout 60s"]
```

Comment:
Can we just simplify this and set the memory limit to 250Mi, and run a single instance of --vm?

Reply:
I see this take a long time to reach OOMKill, as the second stress thread takes a while to ramp up.  Let's just make sure the application can abuse, and abuse in a short amount of time?